packages/legend-application-query/src/application/LegendQueryApplicationConfig.ts

```diff
@@ -70,6 +70,8 @@ class LegendQueryApplicationCoreOptions {
    */
   queryBuilderConfig: QueryBuilderConfig | undefined;
 
+  applicationName = 'Legend Query';
```

Comment:
is this really needed or can't we just load the query-builder header with the panel having this?